src/test/java/com/google/api/codegen/testdata/go_library.baseline

```diff
@@ -635,54 +635,54 @@ func (it *StringIterator) NextPageToken() string {
     return it.nextPageToken
 }
 ============== file: google.golang.org/cloud/library/generated/library/client_test.go ==============
-package library
+package library_test
 
 import (
     gax "github.com/googleapis/gax-golang"
     "golang.org/x/net/context"
     google_example_library_v1 "google.golang.org/cloud/library/generated/google/example/library/v1"
```

Comment:
Gotcha. If I'm not mistaken, the proto compiler chooses the last component of the `go_package` string, so in this case it chooses `v2`.

I'm not sure if there is another option that can override the actual package name... I haven't seen it yet.